### DISCUSSION TITLE
MixxxPiBeta64v2

### DISCUSSION THREAD

**Discussion by fayaaz** [fayaaz/mixxx-pi-gen#30] [General]
- AAC support enabled
- KeyFinder enabled again
- Mixxx 2.3 no longer in beta version:  [d585e26436f456a51c18e53e35ac3d421af251b3](https://github.com/mixxxdj/mixxx/commit/d585e26436f456a51c18e53e35ac3d421af251b3)

<hr /><em>This discussion was created from the release <a href='https://github.com/fayaaz/mixxx-pi-gen/releases/tag/'>MixxxPiBeta64v2</a>.</em>

**Comment by camplusplus** [comment]
I seem to have white waveform when glsl / rgb is selected by default but when changed to gl it works. Latency is really good with my usb controller sound card (11ms). Keep the good work!

**Reply by fayaaz** [reply]
Are you using key lock and 2 decks? Glad to hear the performance is good!

**Comment by camplusplus** [comment]
Hi, no key lock

**Comment by hackerb9** [comment]
Just double checking that the release called "Beta" (in "MixxxPiBeta64v2") has the non-beta release of Mixxx-2.3, right? 
(By the way thanks for making this image, it helps out a lot!)

**Reply by fayaaz** [reply]
Yes it's using a 2.3 built version but possibly not the 2.3.1 release.

**Comment by Maxwell215** [comment]
Can you give me 2.3.1 zip on raspberry pi 4 please. I want to use rekordbox library to play music on pi 4

**Comment by camplusplus** [comment]
I use rekordbox on 2.3.0

**Comment by slothstronomic** [comment]
Is there command line access in this?

**Reply by fayaaz** [reply]
This is a Raspberry pi debian image with some customisations. It comes with the standard bash or sh. The default username and password are in the [README](https://github.com/fayaaz/mixxx-pi-gen#readme)

**Reply by slothstronomic** [reply]
thx

**Comment by JaccoR** [comment]
This is not working on raspberry pi 3b, it is booting into tty1 and then stops. After your armhf PR it should be working right?
